torch/library.h

```diff
@@ -218,6 +273,19 @@ inline CppFunction dispatch(c10::DispatchKey k, Func&& raw_f) {
   return f;
 }
 
+// See Note [Plumbing Keys Through The Dispatcher]
+template <typename Func>
+inline CppFunction dispatch_withKeys(c10::DispatchKey k, Func&& raw_f) {
```

Comment:
I think this variant is 100% unnecessary, even ignoring my previous comment. Try deleting it.

Reply:
Alright, I take it back, I see usages of this function. However, with my previous comment, you would then have a CppFunction that "knows" its with keys and then you wouldn't need to set it here.